### ISSUE TITLE
Using quotes when exporting in dotenv format

### ISSUE THREAD

**Issue by w32-blaster** [segmentio/chamber#208]
When using the export command with format dotenv, the value is always in double quote: 
https://github.com/segmentio/chamber/blob/master/cmd/export.go#L120

In some cases this is causing issues. 
For example when the dotenv file (which was exported with chamber) is used to create a new secret in kubernetes. The new secret is created with the environment variables in it, but each of them has the double quote in the value as part of the value string. 
Also, the example in the code doesn't have that quote either: 
https://github.com/segmentio/chamber/blob/master/cmd/export.go#L116

**Comment by nickatsegment** [comment]
I doubt there's a `dotenv` spec, but a quick survey shows that double-quotes are pretty common. So I'm not sure we're doing something wrong here.

What do you mean "creating a new secret in Kubernetes"? I've done this successfully myself with `chamber export` in JSON plus `jq`, e.g.:
```
chamber export myservice | jq '{
  "kind": "Secret",
  "apiVersion": "v1",
  "metadata": {
    "name": "mysecret"
  },
"data": (with_entries(.value = (.value | @base64)))
}'
```

**Comment by w32-blaster** [comment]
I have used this method:
`kubectl create secret generic secret-name --from-env-file=./.env`
The command above will generate a single secret based on the dotenv file. 
On the cluster I tried to use the environment in a nodejs application which seems to be failing because of the double quotes around the values. 

For example when using elastic beanstalk, the EC2 instances are using similar structure of environment variable setup. There I don't see double quotes, so I'm wondering if this is really needed?

**Comment by nickatsegment** [comment]
I did some digging.

We added env file support in https://github.com/segmentio/chamber/commit/1fdd080354c65c8ef429b04dd98938e789830931. It was just `%s=%s\n`. I'm not sure what the motivation was, or if there was a particular dialect we intended to support.

The problem with this naive support is, how do you escape newlines? So we accepted a PR that added quotes in https://github.com/segmentio/chamber/pull/150

I found kubectl's parser at https://sourcegraph.com/github.com/kubernetes/kubernetes@5a7b978c7401efad5672bcc876bcf5d3dfa71bd0/-/blob/pkg/kubectl/generate/versioned/env_file.go#L35:6 It looks pretty naive, and wouldn't support a value with a newline in it. I bet this is based off of Docker's format https://docs.docker.com/compose/environment-variables/. People have proposed allowing quotes to handle the newline case, but this was rejected as not backwards compatible. https://github.com/moby/moby/issues/12997#issuecomment-154180116

TLDR: .env is an ill-defined format, so whatever we do will break somebody's toolchain. So I'm not going to revert it, and I'm afraid you'll have to work around the quotes. The laziest would be something like `kubectl create secret generic secret-name --from-env-file=<(sed 's/"//g' < ./.env)`, but this is going to break as soon as you have a secret with a newline in the value. Probably better to use the JSON export to jq solution I mentioned earlier.

**Comment by gorankarlic** [comment]
This is not OK - quotes in ‘dotenv’ values are quite common. IMHO ‘—from-env-file’ should behave like BSD ‘env(0)’
